pleasew

```diff
@@ -18,6 +18,8 @@ if [ -z "$LOCATION" ]; then
 else
     # It can contain a literal ~, need to explicitly handle that.
     LOCATION="${LOCATION/\~/$HOME}"
+    # It can contain a literal $USER, need to explicitly handle that.
+    LOCATION="${LOCATION/\$USER/$USER}"
```

Comment:
This will replace the literal `$USER` with the value of  the envvar `$USER`, e.g. using the example we've been talking about out of band in `~/.config/please/plzconfig`:

```
[Please]
location = /opt/.../please/$USER
```

and assuming your local user account was `Tatskaari`, `LOCATION` would then become `/opt/.../please/Tatskaari`

Reply:
What i mean is that Please doesn't support this. While this script might expand this var, Please will evaluate this literally which will cause problems later down the line. Is there a reason we need this expanded?

Reply:
Ah I see, I wasn't aware that `location` was also used by please - so without the expansion, pleasew will download the file into a folder literally called `/opt/.../please/$USER`.

Reply:
💡 , ah I understand, the per user config in ~/.config/ means we can hardcode the username making this redundant. I'll remove this line.